3p/iframe-transport-client.js

```diff
@@ -161,6 +165,26 @@ export class IframeTransportContext {
     this.iframeMessagingClient_./*OK*/sendMessage(
         MessageType.IFRAME_TRANSPORT_RESPONSE,
         /** @type {!JsonObject} */
-        (Object.assign({message: data}, this.baseMessage_)));
+        (Object.assign({message: data},
```

Comment:
Why use Object.assign if you're not using an already existing object? Also, you can avoid the cast here by using dict().